lib/customerio.ex

```diff
@@ -801,4 +801,24 @@ defmodule Customerio do
     {:ok, result} -> result
     {:error, e} -> raise e
   end
+
+  @doc """
+  Trigger a campaign execution.
+
+  ## Params
+
+    * `id` - the unique identifier of the campaign.
+
+    * `data` - data to be included in the campaign.
+
+  ## Example
+
+  ```elixir
+  iex> Customerio.trigger_campaign(1, %{data: %{title: "hello"}})
+  ```
+  """
+
```

Comment:
Don't forget to add specs for this function - dializer will make a great job here.

```suggestion

@spec ...
```

Reply:
👍